### ISSUE TITLE
NodeMCU V3 Lua WiFi ESP8266 32M USB CH340G TTL Internet Development Board Module

### ISSUE THREAD

**Issue by soydemadrid** [bozimmerman/Zimodem#18]
Hi I hope you can help.

I'm running Windows Arduino IDE 1.8.5 and added the json for esp8266 boards etc.

I have a NodeMCU V3 Lua WiFi ESP8266 32M USB CH340G TTL Internet Development Board Module from ebay and it compiles and flashes no problem.

But then when I run Kermit on my old dos pc I can only use a limited set of AT commands and can't setup wifi or follow your instructions. I just get "ERROR".

I can do AT and it says OK
I can do ATI01 through to ATI04 but not 05,06,07 onwards.

Is there anything I'm doing wrong or can you help at all? I'm hoping to get my old dos pc on wifi through Kermit!

Thanks for any help :)

**Comment by soydemadrid** [comment]
ps this is the guide I followed which recommended your Zimodem firmware:

http://subethasoftware.com/2018/02/01/wifi-coco-for-15-or-for-any-retro-computer-with-an-rs-232-port/

**Comment by soydemadrid** [comment]
Hi further to this my issue isn't at all what I first thought!! I tried changing my Com port in Kermit and realised the AT commands are working in a limited way due to me accidentally trying to use a built in Modem!! Not the ESP!!

But when I now do get switched to the right ESP8266 serial port, the terminal  just freezes and I can't type. So my issue is an even worse one now...

I flashed initially with the MAX3232 hooked up like how the guide showed me. Could this effect the flash at all? Is it best to flash without the MAX plugged in?
